src/index.js

```diff
@@ -57,7 +57,7 @@ export function app(state, actions, view, container) {
 
   function resolveNode(node) {
     return typeof node === "function"
-      ? resolveNode(node(globalState, wiredActions)) || h(node(globalState, wiredActions), null))
+      ? resolveNode(node(globalState, wiredActions)) || h(node, null)
```

Comment:
`h(node)` same as `h(Component)`. For component which returns null the `h` function will also return `null`.
This line become `null || null`

Reply:
Yes @frenzzy, I just realize that, thank you!